### PR TITLE
ES 1.x gets the ES 2.x template

### LINKED ISSUE #456: Add support for ES 1.x to use the ES 2.x template
This is what's currently being done anyway.

### PR DESCRIPTION
**Pull request by untergeek**
This is the current default behavior anyway

fixes #456

### CODE REVIEW

File: lib/logstash/outputs/elasticsearch/template_manager.rb [hunk 1]
```diff
@@ -30,7 +30,8 @@ def self.install(client, template_name, template, template_overwrite)
     end
 
     def self.default_template_path(es_major_version)
-      default_template_name = "elasticsearch-template-es#{es_major_version}x.json"
+      es_major_version == "1" ? template_version = "2" : template_version = es_major_version
```
Comment (ph) [inline lib/logstash/outputs/elasticsearch/template_manager.rb]: I think you meant this

```ruby
template_version = es_major_version == "1" ?  "2" :  es_major_version
```

I was a bit confused with the assignment in the ternary ops.

**Comment by ph** [comment]
LGTM, I havent tested with an actual ES 1.7 but I am sure it resolve #456

**Comment by untergeek** [comment]
Thanks for fixing the ternary operator.  I'm still trying to cross over my python domain knowledge into Ruby... 😮

**Comment by ph** [comment]
> Thanks for fixing the ternary operator. I'm still trying to cross over my python domain knowledge into Ruby... 😮

Don't worry don't tell anyone I use semicolon in ruby now because of java..

**Comment by ph** [comment]
LGTM

**Pull request closed without merging**